sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1412,7 +1364,7 @@ static int create_sdw_dailink(struct snd_soc_card *card, int *link_index,
 	int stream;
 	int i = 0;
 	int ret;
-	int k;
+	int j, k;
```

Comment:
nitpick: keep one variable per line?

Reply:
Personally I would lean towards these being fine on one line since they both loop counters, so kinda logically grouped. And its not even the only case of multiple variables on a single line in this function, let alone the whole file.

All that said, I also don't greatly mind so if people want j on a separate line happy to make it so.

Reply:
i prefer one line per variable, mainly because diffs are easier to spot.